src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -647,4 +653,85 @@ private String preProcessImageForCsvRow(String csvRow, List<String> imageFileNam
             return CsvMonumentConverter.setImageFileNameOnCsvRow(csvRow, "");
         }
     }
+
+    /**
+     * Gathers the various statistics related to Monuments for the About Page
+     * @return MonumentAboutPageStatistics - Object containing the various statistics relating to Monuments for the
+     * About Page
+     */
+    public MonumentAboutPageStatistics getMonumentAboutPageStatistics() {
```

Comment:
In a perfect world this is something we would cache and only update every day or so, but that's a lot of effort to implement

Reply:
Would be pretty cool though.